release-tools/scripts/manifest.yml

```diff
@@ -148,7 +148,7 @@ plugins:
     plugin_basename: opendistro-asynchronous-search
     plugin_git: opendistro-for-elasticsearch/asynchronous-search
     plugin_version: 1.13.0.0
-    plugin_build: 
+    plugin_build: 20
```

Comment:
The new plugin version is 1.13.0.1 and the build number is 21.

Reply:
Changed!